src/plugins/recordTypes/media/forms/default.jsx

```diff
@@ -0,0 +1,92 @@
+const template = (configContext) => {
+  const {
+    React,
+  } = configContext.lib;
+
+  const {
+    Col,
+    Cols,
+    Panel,
+    Row,
+  } = configContext.layoutComponents;
+
+  const {
+    Field,
+    Subrecord,
+  } = configContext.recordComponents;
+
+  const {
+    extensions,
+  } = configContext.config;
+
+  return (
+    <Field name="document">
+      <Panel name="media" collapsible>
+        <Row>
+          <Field name="identificationNumber" />
+          <Field name="approvedForWeb" subpath="ns2:media_pahma" />
+        </Row>
+        <Field name="description" />
+
+        <Panel name="file" collapsible>
+          <Subrecord name="blob" showDetachButton />
+        </Panel>
+
+        <Field name="externalUrl" />
+
+        {extensions.dimension.form}
+
+        <Cols>
+          <Col>
+            <Field name="contributor" />
+            <Field name="creator" />
+
+            <Row>
+              <Field name="primaryDisplay" subpath="ns2:media_pahma" />
+              <Field name="documentationImage" subpath="ns2:media_pahma" />
+            </Row>
+
+            <Field name="publisher" />
+
+            <Field name="relationList">
+              <Field name="relation" />
+            </Field>
+
+            <Field name="copyrightStatement" />
+          </Col>
+
+          <Col>
+            <Row>
```

Comment:
Wrap the languageList and typeList fields in `Col` elements. This will restore the margin between them. (Workaround for margin being lost between repeating fields in a row).